.github/workflows/ci.yml

```diff
@@ -116,9 +116,3 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           CYPRESS_PROJECT_ID: ${{ secrets.CYPRESS_PROJECT_ID }}
-          # - if the event is push, the title will be undefined
-          # and Cypress will get the commit message from Git information
-          # - if the event is pull_request, then we set the commit
-          # message to the pull request title
-          # https://github.com/cypress-io/github-action/issues/124#issuecomment-653180260
-          COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
```

Comment:
I realized we should be getting some Cypress comments within GitHub PRs with E2E results. It turns out [this property is fixing such comments](https://github.com/cypress-io/github-action/issues/124#issuecomment-716584972), so I'm removing it since 1) we don't need it and 2) will fix the Cy comments.

Reply:
However, we get this ugly title for the run in Cy Dashboard instead:

<img width="829" alt="CleanShot 2021-11-03 at 16 08 11@2x" src="https://user-images.githubusercontent.com/2677072/140087164-939e009a-93be-4ad7-9cfe-967e883a3994.png">

Which option you prefer @HonkingGoose ?

Reply:
Reverting for now since the commit info message seems more useful, and we can still access the Dashboard results from Actions within the PR.